File: ticktacktoe-service/src/ticktacktoe/Entities/GameState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ticktacktoe.Entities
{
    public class GameState
    {
        public Guid ID { get; set; }

        public List<Player> Players { get; set; }

        public Player CurrentPlayer { get; set; }

        public List<string> Board { get; set; }
    }
}

File: ticktacktoe-ui/src/app/services/game-connection/game-state.ts
import { Player } from './player';
import { RoundResult } from '../../types/round-result';

export interface GameState {
    id: string;
    players: Player[];
    currentPlayer: Player;
    board: string[],
    roundResult: RoundResult;
    round: number;
    score: any;
}